src/hooks/useEntry.ts

```diff
@@ -1,6 +1,6 @@
 import { RequestBundle, apiRequest } from './apiRequest';
-
-const ENTRY_BASE_URL = process.env.NEXT_PUBLIC_ENTRY_BASE_URL;
+import { FlashContext } from '../contexts/FlashContext';
+import { useContext } from 'react';
 
```

Comment:
Same suggestion as made in `useAccess.ts`.

```suggestion

const ENTRY_BASE_URL = process.env.NEXT_PUBLIC_ENTRY_BASE_URL;

```